vm/actor_interface/src/lib.rs

```diff
@@ -66,8 +67,8 @@ impl From<StateTreeVersion> for ActorVersion {
             StateTreeVersion::V0 => ActorVersion::V0,
             StateTreeVersion::V1 => ActorVersion::V2,
             StateTreeVersion::V2 => ActorVersion::V3,
-            StateTreeVersion::V3 => ActorVersion::V4,
-            StateTreeVersion::V4 => ActorVersion::V5,
+            StateTreeVersion::V3 => ActorVersion::V6,
+            StateTreeVersion::V4 => ActorVersion::V6,
             //StateTreeVersion::V4 => ActorVersion::V6, // TODO: what version should it point to?
```

Comment:
I guess this mystery is already solved?

Reply:
This is related to state migration / versioning in the native backend. We definitely used the wrong versions in the past but not in a way that mattered. Thankfully all of this code will be deleted soon.